pkgs/by-name/no/notion-app/update/update.mjs

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env nix-shell
+/*
+#!nix-shell -i node --pure --packages cacert nodejs yq-go
+*/
+import * as assert from "node:assert/strict";
+import * as child_process from "node:child_process";
+import * as fsPromises from "node:fs/promises";
+import * as path from "node:path";
+
+const __dirname = import.meta.dirname;
+
+/** @typedef {{
+  version: string;
+  path: `${string}.zip`;
+  sha512: string;
+  releaseDate: string;
+  files: Array<{ url: string; sha512: string; size: number }>;
+}} LiveCheckInfo */
+
+/** @typedef {{
+  version: string;
+  url: string;
+  hash: `sha512-${string}`;
+}} Info */
+
+/** @typedef {{
+  "x86_64-darwin": Info;
+  "aarch64-darwin": Info;
+}} InfoMap */
+
+const BASE_URL = "https://desktop-release.notion-static.com/";
+/**
+ *
+ * @param {"latest-mac.yml" | "arm64-mac.yml"} liveCheckFile
+ * @returns {Promise<Info>}
+ */
+async function getInfo(liveCheckFile) {
+  const url = /** @type {const} */ (`${BASE_URL}${liveCheckFile}`);
+  const response = await fetch(url);
+  /** @type {LiveCheckInfo} */
+  const { version, path, sha512 } = JSON.parse(
+    child_process
+      .execSync(`yq eval --output-format=json`, {
```

Comment:
@lucasew Is there any simple way to use npm package to parse yaml? Calling yq  executebale looks weird.

Reply:
Not in the standard lib AFAIK. Yeah, it's a little weird but I think it's fine.